README.md

```diff
@@ -426,12 +426,14 @@ await service.edit({
 ### Custom Request Libraries
 
 There is another constructor parameter that allows the user to specify their own custom request library
-as long as it has a similar API to got. To specify the library, simply set the `requester` property when
+as long as it has a similar API to ky. To specify the library, simply set the `requester` property when
 instatiating a service:
 
+An example can be seen in the [KyRequester.ts](https://github.com/jdalrymple/node-gitlab/blob/next/src/infrastructure/KyRequester.ts) file
```

Comment:
```suggestion
An example can be seen in the [KyRequester.ts](./src/infrastructure/KyRequester.ts) file
```
Will work from readme and will work once merged to master :sweat_smile:

Reply:
I didnt know i could do that!! I was just going to manually fix it after lol

Reply:
Yup that works, even looking on the commit itself it resolves to `https://github.com/jdalrymple/node-gitlab/blob/01f65a00cd48592c946a98182e0cffdb23dbcb17/src/infrastructure/KyRequester.ts`